### DISCUSSION TITLE
Public API usage guidelines

### DISCUSSION THREAD

**Discussion by apwelsh** [iopool/community#8] [General]
Can you please provide the usage guidelines for the public API?  I am creating an integration for Hubitat over the public API and I want to hard-code any limitations that need to be in place, as a safety measure. 

For instance, how often (frequency) may I poll the public API?  Every 1 minute, every 5 minutes?  1440 times a day, but no interval limit?  I need to know these restrictions to prevent users from negatively impacting the service.

Thanks.

**Comment by mehdyouras** [comment]
The limitation is 1 request per second.
